distribution/helm/vector-agent/values.yaml

```diff
@@ -148,12 +148,57 @@ psp:
 # of using a generated one.
 existingConfigMap: ""
 
+# Specify custom contents for the Vector config
+## ref: https://vector.dev/docs/reference/configuration/
+## Note a complete and valid configuration is required. If used, the deprecated
+## configuration keys will be ignored. More information can be found at:
+## https://vector.dev/highlights/2021-07-13-helm-customconfig
```

Comment:
Nit: is the `##` intentional here?

Reply:
Copied the style over from Datadog, IIRC it may be related to tooling that helps to generate readme pages...
Ex: https://github.com/DataDog/helm-charts/blob/master/charts/datadog/values.yaml#L173-L179

Reply:
Ah, gotcha, yeah I could see that. I guess I'd lean towards just doing `#` until we have similar tooling.